### ISSUE TITLE
Update widget CI infrastructure to handle tests with Laxar 1.0.0

### ISSUE THREAD

**Issue by jpommerening** [LaxarJS/grunt-laxar#91]
(See LaxarJS/ax-developer-tools-widget#30)

Currently, widget tests on Travis tend to fail because Bower is unable to satisfy concurring versions of Angular/Laxar/Laxar UIKit.
The reason behind this is that we first install the Bower dependencies as they are specified in the widget's `bower.json` file and only _then_ do we try to install older versions of LaxarJS. Bower then tries to satisfy the recursive dependencies of both, the already installed LaxarJS version, and the version we are trying to install "manually". This often fails due to incompatible AngularJS versions.

Solution: Update the versions inside `bower.json` file before installing _anything_, then install without any intervention and run tests.

**Comment by jpommerening** [comment]
Currently the widgets' `.travis.yml` files usually contain:

```yml
install:
   - # <other stuff>
   - npm install
   - bower install "laxar#${LAXAR_VERSION}"
```

Where `${LAXAR_VERSION}` is a variable exported in the `matrix` section of the build.
The test infrastructure has been updated to read the versions from these environment variables. For any dependency inside the `bower.json` file, one can override the version by exporting an environment variable named like the upper-cased, underscore-separated dependency name plus a `_VERSION` suffix. For example, to override the version of `laxar-uikit` one would export `LAXAR_UIKIT_VERSION="..."`.

### Upgrade info

Widgets _can_ now drop the `bower install "laxar#${LAXAR_VERSION}"` line from their `.travis.yml` file. Any version overrides are now handled by `npm install`. To override the version of a dependency declared in `bower.json`, an environment variable (see above for details) needs to be exported during `npm install`.

**Comment by jpommerening** [comment]
Fixed on `master` (`v1.2.0`) & released as `v1.0.4`, `v1.1.1`
